src/postgres/commands/createPostgresServer/IPostgresServerWizardContext.ts

```diff
@@ -3,24 +3,24 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
+import { Sku } from "@azure/arm-postgresql/esm/models";
 import { Server } from "@azure/arm-postgresql/src/models";
```

Comment:
Is there a reason `Sku` was imported from `esm/models`? If not, may as well use the `src/models` one

```suggestion
import { Server, Sku } from "@azure/arm-postgresql/src/models";
```